src/app/static/src/store/getters.ts

```diff
@@ -21,5 +22,11 @@ export const getters = {
         // NB counting from today for now, but this will be updated to count from last reporting
         // day in country data in mrc-2442
         return dayjs().startOf("day").add(forecastDays, "days").toDate();
+    },
+    population: (state: RootState): string => {
+        const population = state.countries && state.countries
+            .find((country) => country.code === state.paramValues!.region)!.population;
+
+        return numericFormatter(population || 0);
```

Comment:
There is absolutely no need to change this, but just to note that comet uses a sufficiently new enough version of TypeScript to allow the following (i.e. both `?.` and `??` operators):
```suggestion
        return state.countries?.find((country) => country.code === state.paramValues!.region)!.population ?? 0;
```
`??` is strictly more appropriate here because `0` is obviously a valid number even though falsely, so doesn't need to be replaced with `0`, as `||` would do.